packages/server-wallet/src/config.ts

```diff
@@ -49,4 +52,21 @@ const config: ServerWalletConfig = {
   metricsOutputFile: process.env.METRICS_OUTPUT_FILE,
 };
 
-export default config;
+export const payerConfig: ServerWalletConfig = {...defaultConfig, postgresDBName: 'payer'};
+export const receiverConfig: ServerWalletConfig = {...defaultConfig, postgresDBName: 'receiver'};
```

Comment:
These seem like test constructs. Do we want to export them from a `src` file?

Reply:
dc74ccd